test/url.jsx

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai';
+import React from 'react';
+
+import { url } from '../';
+
+import callValidator from './_callValidator';
+
+describe('url', () => {
+  const validUrl = 'https://www.airbnb.com';
+  const invalidUrl = 'foobar';
+  const propName = 'urlProp';
+  const componentName = 'url test';
+
+  it('returns a function', () => {
+    expect(typeof url()).to.equal('function');
+  });
+
+  function getElementWithUrlPropValue(value) {
+    return (<div urlProp={value} />);
+  }
+
+  function callUrlPropTypeValidator(element) {
+    return callValidator(url(), element, propName, componentName);
```

Comment:
also, it'd be nice here to follow the other tests, and repeat the `url()` usage in each test, and pass the validator into the `call` and `assert` functions

Reply:
gotcha! let me know if it's still too helper-function-happy.